lib/tilex_web/schema/post_types.ex

```diff
@@ -0,0 +1,24 @@
+defmodule TilexWeb.Schema.PostTypes do
+  use Absinthe.Schema.Notation
+  use Absinthe.Ecto, repo: Tilex.Repo
+
+  object :post do
+    field(:title, :string)
+    field(:body, :string)
+    field(:slug, :string)
+    field(:likes, :integer)
+    field(:developer, :developer, resolve: assoc(:developer))
```

Comment:
logic here for `developer` and `channel` should probably change to use DataLoaders https://hexdocs.pm/absinthe/dataloader.html

Reply:
yea, its definitely not in s state that should be reviewed at the moment. I don't even thing absinthe_ecto is around anymore.